modules/tls-app/README.md

```diff
@@ -0,0 +1,17 @@
+# TLS app Terraform module
+
+This module defines the ACLs and quota needed by an app to communicate with an mTLS enabled kafka cluster.
+
+See [main.tf](main.tf) for what resources it defines and [variables.tf](variables.tf) for available input variables.
+
+## Usage example:
+```terraform
+module "example" {
+  source           = "../../modules/tls-app"
+  consume_topics   = { "example-topic1" : "example-consumer-group1", "example-topic2": "example-consumer-group2" }
+  produce_topics   = ["example-produce-topic"]
```

Comment:
does this conflict with the usage guidelines?

> prefix the names with the name of your team

Reply:
Well, this is meant to be used everywhere, not only on the shared kafka cluster, once the teams will start replacing proximo with tls enabled kafka clusters.
But I feel that this is very very very far away, and the common use case now will be for migrating to the shared kafka cluster.
I'm gonna put 2 examples in this case.